packages/lsp11-contracts/package.json

```
@@ -0,0 +1,52 @@
+{
+  "name": "@lukso/lsp11-contracts",
+  "version": "0.0.1",
+  "description": "Package for the LSP11 Social Recovery standard",
+  "license": "Apache-2.0",
+  "author": "",
+  "main": "./dist/index.cjs",
+  "module": "./dist/index.mjs",
+  "typings": "./dist/index.d.ts",
+  "exports": {
+    ".": {
+      "require": "./dist/index.cjs",
+      "import": "./dist/index.mjs",
+      "types": "./dist/index.d.ts"
+    },
+    "./artifacts/*": "./artifacts/*",
+    "./package.json": "./package.json"
+  },
+  "files": [
+    "contracts/**/*.sol",
+    "!contracts/Mocks/**/*.sol",
+    "artifacts/*.json",
+    "dist",
+    "./README.md"
+  ],
+  "keywords": [
+    "LUKSO",
+    "LSP",
+    "Blockchain",
+    "Standards",
+    "Smart Contracts",
+    "Ethereum",
+    "EVM",
+    "Solidity"
+  ],
+  "scripts": {
+    "build": "hardhat compile --show-stack-traces",
+    "build:foundry": "forge build",
+    "build:js": "unbuild",
+    "clean": "hardhat clean && rm -Rf dist/",
+    "format": "prettier --write .",
+    "lint": "eslint . --ext .ts,.js",
+    "lint:solidity": "solhint 'contracts/**/*.sol' && prettier --check 'contracts/**/*.sol'",
+    "test:foundry": "FOUNDRY_PROFILE=lsp11 forge test --no-match-test Skip -vvv",
+    "test:coverage": "hardhat coverage"
+  },
+  "dependencies": {
+    "@erc725/smart-contracts": "^7.0.0",
+    "@lukso/lsp25-contracts": "*",
```

Need to specify the version now: 

```suggestion
    "@lukso/lsp25-contracts": "~0.15.0-rc.1",
```